config/Variables.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is the output of the processed variables file, we need to have original file for the envsubst to work